### PR TITLE
[documentation] document review timeout

### LINKED ISSUE #112: no timeout of reviewing to ask another reviewer is not defined in the procedure
##### ISSUE TYPE

 - Documentation Report

##### ROLE VERSION

```
N/A
```

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

N/A

##### SUMMARY

no timeout of reviewing to ask another reviewer is defined in https://github.com/reallyenglish/ansible-role-example/tree/master/docs/Procedures/Reviewing_Pull_Request.

##### STEPS TO REPRODUCE

ask a review and get no response from the reviewer.

##### EXPECTED RESULTS

timeout of reviewing to ask another reviewer is defined in https://github.com/reallyenglish/ansible-role-example/tree/master/docs/Procedures/Reviewing_Pull_Request.

##### ACTUAL RESULTS

no timeout  to ask another reviewer is defined in https://github.com/reallyenglish/ansible-role-example/tree/master/docs/Procedures/Reviewing_Pull_Request.

### PR DESCRIPTION
**Pull request by trombik**
fixes #112

### CODE REVIEW

**Comment by trombik** [comment]
this PR would introduce changes in the rules that all members should be aware of.

File: docs/Procedures/Reviewing_Pull_Request/README.md [hunk 1]
```diff
@@ -26,6 +26,20 @@ When creating a PR, the author of the PR must request review, enforced by
...
+
+* the PR is not reviewed within 24 hours, excluding weekdays and public
+  holidays, and no response is given from the reviewer
+* the PR is not reviewed within 48 hours, excluding weekdays and public
+  holidays
+
+When the PR needs to be merged urgently, such as fixing a show-stopper in
+development or deployment, and the author cannot wait the review longer than
+above limits, the author must notify the assignee of the reason to change
+assignee and may assign the PR to others. In this case, it is strongly
```
Comment (mitsururike) [inline docs/Procedures/Reviewing_Pull_Request/README.md]: there is no description about requesting urgent review, which should be before this paragraph.

Comment (trombik) [inline docs/Procedures/Reviewing_Pull_Request/README.md]: do you mean this paragraph should be placed earlier?

Comment (mitsururike) [inline docs/Procedures/Reviewing_Pull_Request/README.md]: i mean a paragraph describing `urgent review` should be added before the paragraph.

Comment (trombik) [inline docs/Procedures/Reviewing_Pull_Request/README.md]: does 4db290f fix what you have requested?

Comment (mitsururike) [inline docs/Procedures/Reviewing_Pull_Request/README.md]: better but no.
* `request a review`
* `define urgent PR and how to notify the reviewer`
* `review timeout`
  - `if it is an urgent PR`

Comment (trombik) [inline docs/Procedures/Reviewing_Pull_Request/README.md]: after a long thought, i still do not get it. what does the first list item mean in the documentation for reviewers?

do you think "When the PR needs to be merged urgently, such as fixing a show-stopper in development or deployment" is not enough definition?

why is the last item at second level?

Comment (mitsururike) [inline docs/Procedures/Reviewing_Pull_Request/README.md]: the level is not essential.
if the PR is urgent, the reviewer should be notified when requested.
it seems the reviewer is not notified until the shorter timeout in your explanation.

* ask a review when PR
* if it is urgent, notify the reviewer that it is urgent
* timeout of review
* if it is urgent, the timeout may be shorter than usual.

File: docs/Procedures/Asking_Review_and_Getting_Feedback/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+![Asking Review and Getting Feedback](images/Asking_Review_and_Getting_Feedback.png)
+
+## Making sure all the tests pass
+
+PRs must pass all the tests. See the build status in the PR.
+
+## Assigning Reviewer
+
+When creating a PR, the author of the PR must request review, enforced by
+settings in project and role repositories. Merging PRs is blocked until at
```
Comment (mitsururike) [inline docs/Procedures/Asking_Review_and_Getting_Feedback/README.md]: it is not configured to be blocked in the procedure `docs/Procedures/Maintaining_Repository`.

File: docs/Procedures/Creating_Pull_Request/README.md [hunk 3]
```diff
@@ -89,64 +88,40 @@ Examples:
...
-
-It is not mandatory but recommended to notify the reviewers. Github sends
-notifications to the reviewers but it is always a good idea to make sure your
-PR is not forgotten.
-
 ## Fixing or correcting the PR
 
 When the review finishes, and the reviewer leaves comments and/or asks
 modifications, change the code or discuss with the reviewer. Repeat until the
 reviewer approves. Should the discussion stacks, ask the Project Owner.
```
Comment (mitsururike) [inline docs/Procedures/Creating_Pull_Request/README.md]: who is the `Project Owner` for ansible-role-*?

File: docs/Procedures/Asking_Review_and_Getting_Feedback/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  holidays
+* the deadline the author has set is missed
+
+## Fixing the branch
+
+When the PR is reviewed and the PR is not approved, fix the branch with the
+feedback or discuss with the reviewer. Repeat the process from [Making sure all
+the tests pass].
+
+Should the discussion stacks, ask the Project Owner.
```
Comment (mitsururike) [inline docs/Procedures/Asking_Review_and_Getting_Feedback/README.md]: who is the `Project Owner` of ansible-role-*?

File: docs/Procedures/Asking_Review_and_Getting_Feedback/README.md (hunk 2 above)
Comment (trombik) [inline docs/Procedures/Asking_Review_and_Getting_Feedback/README.md]: would you create an issue? thanks

**Pull request merged**